src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/dcoff.cpp

```diff
@@ -209,6 +209,48 @@ void finalize_remap(Function& fbody, Subgraph& fsg, const ClosureRemap& m) {
     }
 
     fbody._model->validate_nodes_and_infer_types();
+
+    struct QuantUnpackGatherParams {
```

Comment:
Might be the same structure as here: https://github.com/openvinotoolkit/openvino/pull/30553/files#diff-3d66ecaf9eef476ac8e72cb7cd10ad1c42ba79bdb228bd716bf51d4588a2ed25R65 would be beneficial for visible mapping?

Reply:
It's different order in several files. I'm keeping in the same order in each file as `host_gather`